st/protocol

```diff
@@ -38,11 +38,22 @@ if which wdiff 2>/dev/null ; then
 fi
 
 proto_current="$M0_SRC_DIR/xcode/protocol.txt"
-proto_dev="/tmp/protocol-dev.txt"
-
-echo "Downloading latest protocol file from dev"
-wget -O "$proto_dev" \
-     '10.230.45.149:8080/job/motr-test-dev/lastSuccessfulBuild/artifact/artifacts/protocol.txt'
+proto_prod="http://cortx-storage.colo.seagate.com/releases/cortx/github/main/"
+proto_prod+="rockylinux-8.4/last_successful_prod/cortx_iso/"
+test_dir="/tmp/37protocol-test/"
+proto_dev="${test_dir}usr/share/motr/protocol.txt"
+
+# remove old st dir if already exists
+rm -rf $test_dir 2>&1> /dev/null
+
+# create st dir if does not exists
+mkdir -p $test_dir
+
+pushd $test_dir
+echo "Downloading latest protocol file from prod"
+wget -q -np -nd -r -P $test_dir -A "cortx-motr-[0-9]*_git*.x86_64.rpm" $proto_prod
+rpm2cpio cortx-motr*.rpm | cpio -id
+popd
```

Comment:
After line 59 when you are done using the protocol file remove the folder and its contents which you had created on line 50.

Reply:
Yes, but we get exit code of diff command and report it back. So I added this part at line #47 please check if that is ok.

Reply:
Line # 47 can stay in the code as that is for extreme measures; but we should always do a cleanup of the stuff which we have created. 
Just imaging you run this ST and it creates these folders with the files and thus takes a lot of space in the filesystem. Now another different ST (or some other program) is started but they do not find enough space to work because of the leftover from this ST. It is not correct to stop some other programs because we did not do our cleanup.

Reply:
Line # 47 can stay in the code as that is for extreme measures; but we should always do a cleanup of the stuff which we have created.
Just imagine you run this ST and it creates these folders with the files and thus takes a lot of space in the filesystem. Now another different ST (or some other program) is started but they do not find enough space to work because of the leftover from this ST. It is not correct to stop some other programs because we did not do our cleanup.